site/content/faq.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Frequently Asked Questions"
+menu = "main"
+menuTitle = "FAQs"
+weight = 550
+pre = '<i class="far fa-question-circle"></i> '
```

Comment:
what's this?

Reply:
It is an icon. This is not used with the current theme "material-docs", but will be shown in theme "learn". I can remove it here and add it after the merge of #1111.